.golangci.yaml

```diff
@@ -151,3 +151,7 @@ linters-settings:
       - name: var-naming
         # Disable because it conflicts with protobuf naming convention like "Id"
         disabled: true
+      - name: redefines-builtin-id
+        # Disabled as the use of identifiers similar to built-ins is intentional
+        # and does not cause any runtime or functional issues.
+        disabled: true
```

Comment:
What were some specific things that were getting flagged?

Reply:
As these are easily fixed, I'd prefer seeing us do so instead of disabling this linter. By disabling it, we're stopping it from finding _future_ problems that might be more genuinely troublesome than these are.